cli/internal/cmd/terminate.go

```diff
@@ -37,7 +37,10 @@ func NewTerminateCmd() *cobra.Command {
 // runTerminate runs the terminate command.
 func runTerminate(cmd *cobra.Command, args []string) error {
 	fileHandler := file.NewHandler(afero.NewOsFs())
-	spinner := newSpinner(cmd.ErrOrStderr())
+	spinner, err := newSpinnerOrStderr(cmd)
```

Comment:
Should we remove the newSpinner funciton so this cannot happen again? The code could just be inlined into the newSpinnerOrStderr function.

Reply:
I checked this. `newSpinner` is used quite a few times in `spinner_test` as well. Inlining this everywhere is probably not the right call. 

The cleaner solution IMO would be to have spinner in its own package and export only required functions.

Reply:
I will refactor this in another PR.